app/views/results/student/_student_summary.html.erb

```diff
@@ -26,6 +26,7 @@
       </div>
     </div>
 
+    <% if (@old_result.get_positive_extra_points != 0 or @old_result.get_negative_extra_points != 0) %>
```

Comment:
Don't use parentheses around the condition. Also, use `||` instead of `or` (these are actually different).